portal-web/docroot/html/taglib/ui/input_localized/page.jsp

```diff
@@ -47,6 +48,14 @@
 
 						inputLocalized.updateInputLanguage(editor.getHTML());
 					}
+
+					function <%= namespace + randomNamespace %>onInitEditor() {
+						var inputLocalized = Liferay.component('<%= namespace + HtmlUtil.escapeJS(fieldName) %>');
```

Comment:
Did you try with the `componentReady` API instead? Here `inputLocalized` could be `null` or `undefined`.

```javascript
Liferay.componentReady('<%= namespace + HtmlUtil.escapeJS(fieldName) %>').then(inputLocalized => {
    inputLocalized.updateInputPlaceholder(window['<%= namespace + HtmlUtil.escapeJS(inputEditorName) %>']);
});
```

Reply:
In fact, if you check `ckeditor.jsp` you can see that:

```html
<c:if test="<%= Validator.isNotNull(onInitMethod) %>">
	window['<%= HtmlUtil.escapeJS(onInitMethod) %>']();
</c:if>

Liferay.component('<%= name %>', window['<%= name %>'], {
	portletId: '<%= portletId %>',
});
```

So, there's a clear chance to run `initEditor` before `component has been registered

Reply:
You're right, let me update this